docs/troubleshooting.rst

```diff
@@ -131,6 +129,24 @@ In general, your options for accounts are:
 .. Warning:: Don't store real value in an account until you are familiar with security best practices.
    If you lose your private key, you lose your account!
 
+Why doesn't my transaction work on another network?
+---------------------------------------------------
+
+web3.py is an Ethereum-specific library, which defaults to
+`"type 2" transactions <https://ethereum.org/en/developers/docs/transactions/#typed-transaction-envelope>`_
```

Comment:
nit: Since the link provided has no mention of "type 2" but does mention EIP-1559, linking the two might be a nice touch.

```suggestion
`"type 2" EIP-1559 transactions <https://ethereum.org/en/developers/docs/transactions/#typed-transaction-envelope>`_
```